src/lib/components/ResponseMultipleChoice/index.js

```diff
@@ -0,0 +1,185 @@
+import {html} from "lit-element";
+import {BaseResponseElement} from "../BaseResponseElement";
+import "./_styles.scss";
+
+/* eslint-disable require-jsdoc */
+class ResponseMultipleChoice extends BaseResponseElement {
+  static get properties() {
+    return {
+      id: {type: String, reflect: true},
```

Comment:
Every element has an ID, so I don't think you need this. The point here is that Lit will trigger a rerender if one of these properties changes. Do you need to rerender when the ID changes?